elements/rh-button-group/rh-button-group.css

```
@@ -3,8 +3,5 @@
   flex-wrap: wrap;
   gap: var(--rh-space-md, 8px) var(--rh-space-lg, 16px);
   margin-inline-end: var(--rh-space-lg, 16px);
-  margin-bottom: var(
-    --rh-space-md,
-    16px
-  );
+  margin-bottom: var(--rh-space-md, 8px);
```

```suggestion
  margin-block-end: var(--rh-space-md, 8px);
```

https://developer.mozilla.org/en-US/docs/Glossary/Logical_properties